### PR TITLE
new-porag

### PR DESCRIPTION
**Pull request by ismanovaazalia**

### CODE REVIEW

**Comment by ismanovaazalia** [comment]
привет

**Comment by ismanovaazalia** [comment]
пока

**Comment by ismanovaazalia** [comment]
екрдзге6ы4р

File: project1/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
       
      <div class="item1">
          <p>Привет, мир</p>
+         <p>Привет Эльзар.Как дела</p>
```
Comment (ghost) [inline project1/index.html]: Let's make it looks like standart
 div class="item2"
here is gonna be your chagnes
div

Comment (ismanovaazalia) [reply project1/index.html]: i'll fix that

**Review by ghost — APPROVED** [review]

**Pull request merged**